zerver/lib/thumbnail.py

```diff
@@ -411,7 +411,7 @@ class MarkdownImageMetadata:
     transcoded_image: StoredThumbnailFormat | None = None
 
 
-def get_user_upload_previews(
+def generate_user_upload_previews(
```

Comment:
My only hesitation is that this name implies that this is the _primary_ way in which thumbnails are enqueued.  In normal state, this enqueuing is duplicative, because there's already one in the queue from when the file was uploaded.  It's only corner cases when this actually enqueues any generation.

Reply:
- `manifest_and_get_user_upload_previews`
- `enqueue_and_get_user_upload_previews`
- ...or `generate_user_upload_previews` and a big fat comment at the top that the default `enqueue` is going to be duplicative.